src/drivers/St7789.cpp

```diff
@@ -62,7 +65,19 @@ void St7789::ColMod() {
 
 void St7789::MemoryDataAccessControl() {
   WriteCommand(static_cast<uint8_t>(Commands::MemoryDataAccessControl));
+// P8B Mirrored version sets MX bit high in MADCTL register to flip about vertical axis.
+#ifdef DRIVER_DISPLAY_MIRROR
+  // [0 .. 1] = Unused
+  // [2] = MH = Display Data Latch Order, 0 = LCD refresh from left to right, 1 = Right to left
+  // [3] = RGB = RGB/BGR Order, 0 = RGB, 1 = BGR
+  // [4] = ML = Line Address Order, 0 = LCD refresh from top to bottom, 1 = Bottom to top
+  // [5] = MV = Page/Column Order, 0 = Normal mode, 1 = Reverse mode
+  // [6] = MX = Column Address Order, 0 = Left to right, 1 = Right to left
+  // [7] = MY = Page Address Order, 0 = Top to bottom, 1 = Bottom to top
+  WriteData(0b01000000);
+#else
```

Comment:
I think the order should be MSB first. Also the top comment may not be necessary, as this information can be deciphered from `DRIVER_DISPLAY_MIRROR` and the written data.

Reply:
Okay, I adjusted the comments.